src/components/layout/CommentInput.jsx

```diff
@@ -8,28 +8,37 @@ const CommentInput = () => {
   const [message, setMessage] = useState('');
   const [focus, setFocus] = useState(false);
 
-  const submitComment = (event) => {
-    event.preventDefault();
+  const submitComment = () => {
     if (subscriber) {
       const comments = {
-        body: event.target.body.value,
+        body: input,
       };
-      Articles.createComment(comments, article.id);
-      setInput('');
-      setMessage('');
-      setFocus(false);
+      if (input.trim().length !== 0) {
+        Articles.createComment(comments, article.id);
+        setInput('');
+        setMessage('');
+        setFocus(false);
+      } else {
+        setMessage("Comment field can't be empty");
+      }
     } else {
       setMessage('Please subscribe to comment');
     }
   };
 
+  const submitCommentHandler = (event) => {
+    if (event.key === 'Enter') {
+      submitComment();
+    }
+  };
+
```

Comment:
Did you get this to work? :) Was it the capital E?

Reply:
Yeah E was the problem here 😅